packages/netlify-cms-core/src/backend.ts

```diff
@@ -702,18 +702,16 @@ export class Backend {
   async persistEntry({
     config,
     collection,
-    entryDraft,
+    entryDraft: draft,
     assetProxies,
     usedSlugs,
     unpublished = false,
     status,
   }: PersistArgs) {
-    const newEntry = entryDraft.getIn(['entry', 'newRecord']) || false;
+    const modifiedData = await this.invokePrePushEvent(draft.get('entry'));
```

Comment:
We should document this so there is no confusion, e.g.
The docs should show:
```js
CMS.registerEventListener({
  name: 'prePush',
  handler: ({ entry }) => {
    return entry.get('data').set('title', 'new title');
  },
});
```
To avoid someone doing:
```js
CMS.registerEventListener({
  name: 'prePush',
  handler: ({ entry }) => {
    return entry.set('path', 'new-path');
  },
});
```